README.md

```diff
@@ -26,6 +45,10 @@ WebAuthn (Web Authentication) is a W3C standard for secure public-key authentica
 - WebAuthn [article with talk](https://developers.google.com/web/updates/2018/05/webauthn) in Google Developers
 - How to use [WebAuthn in Android apps](https://developers.google.com/identity/fido/android/native-apps)
 
+### Why WebAuthn in my web server?
+
+- [Security Benefits for WebAuthn Relying Parties](https://www.w3.org/TR/webauthn/#sctn-rp-benefits)
+
```

Comment:
This feels redundant with the list above, could be left out.

Reply:
I am keeping the link but mixing the subsections.